packages/main/src/types/BarDesign.js

```diff
@@ -0,0 +1,47 @@
+import DataType from "@ui5/webcomponents-base/dist/types/DataType.js";
+
+/**
+ * @lends sap.ui.webcomponents.main.types.BarDesign.prototype
+ * @public
+ */
+const BarTypes = {
+	/**
+	 * Default type
+	 * @public
+	 * @type {BottomBorder}
+	 */
+	BottomBorder: "BottomBorder",
```

Comment:
I know I proposed the names of the design types, but I now I started hesitating, because it is not just the border, but the bg-color, box shadow, border-radius that also change - pehaps "Header", "Footer" and "Floating" are more appropriate.

Reply:
Should we make a brainstorming meeting or just review meeting of the pros and cons of both ways? And to take a action after that or we should just remove the current and take the designs from openui5.

Reply:
I think now that it is part of the fiori package, we can stick to the Fiori names, let's take them - Header, Footer, FloatingFooter